doc/ref_arch/kubernetes/chapters/chapter02.md

```diff
@@ -249,6 +248,7 @@ Please note that "shared" is a reference to multi-tenant support and pooled stor
 | Ref # | sub-category | Description |
 |---|---|---|
 | `req.int.api.02` | API | The Architecture **must** leverage the Kubernetes APIs to discover and declaratively manage compute (NFVI and bare metal resources), network, and storage. |
+| `req.kcm.04` | API | The Architecture **must** support the usage of a Kubernetes Application package manager using the Kubernetes API-s, like Helm v3. |
```

Comment:
Can the reference be, e.g. `req.int.api.02` in line with the rest of the table?

Reply:
Ah sorry that should be `req.int.api.03` as the one above is already `02`.

Reply:
fixed